src/watcher.js

```diff
@@ -33,7 +34,8 @@ export default function onSourceChange(
 export function proxyFileChanges(
     {artifactsDir, onChange, filePath, shouldWatchFile}: Object) {
   if (!shouldWatchFile) {
-    const fileFilter = new FileFilter();
+    const fileFilter = new FileFilter({
+      filePathsToIgnore: [path.resolve(artifactsDir)]});
```

Comment:
I don't think it's necessary to construct a default filter that ignores artifact directories here. The build command already takes responsibility for this.